puppet/precip/manifests/init.pp

```diff
@@ -98,6 +99,23 @@
 
   # Install Memcached
   class { 'memcached': }
+  
+  # Install Elasticsearch
+  # (Allows VM host access and enables CORS for all .vm hosts)
+  class { 'elasticsearch':
```

Comment:
Two stylistic items here:

1) Two Spaces instead of tabs, please: http://puppet-lint.com/checks/hard_tabs/
2) Line up arrows under the config element: http://puppet-lint.com/checks/arrow_alignment/

Reply:
Additionally, rather than just adding Elasticsearch by default, I'd like to see it wrapped in a feature-switch. So, new object in `config.rb`, `Vagrantfile` and a new Fact, tracking enabled optional things. We can then use the same kind of functionality to optionally provision Solr, or optionally provision PML, or optionally provision in-box theme compilation tools.

I'd also like to see any optional things pulled into their own class in their own Puppet file.